### ISSUE TITLE
How to properly cite?

### ISSUE THREAD

**Issue by HamzahNizami** [AlexeyAB/darknet#2782]
Hi there, just wanted to quickly ask about how to properly cite everything I used from this GitHub in my research paper. 

Thanks in advance.

**Comment by cenit** [comment]
@alexeyab what about creating a zenodo doi? It should be the best answer to this issue imho

**Comment by AlexeyAB** [comment]
@cenit 

I have registered in the Zendo. So after switching to OpenCV 4.0, I will create a new release and a link will appear in the Zendo Doi.

Which citation option is preferable, can you give an example?

```
@misc{alexeyab84_darknet,
  author = {Redmon, J., Bochkovskiy, A., Sinigardi, S.},
  title = {Darknet: Yolov3 - neural network for object detection},
  year = {2019},
  publisher = {GitHub},
  journal = {GitHub repository},
  howpublished = {\url{https://github.com/AlexeyAB/darknet}},
  commit = {6231b748c44e2007b5c3cbf765a50b122782c5a2}
}
```

-----

After that I will create the new Release of Darknet there will be something like this:
```
@misc{alexeyab84_darknet_2019_1....,
    author = {Redmon, J., Bochkovskiy, A., Sinigardi, S.},
    title = {Darknet: Yolov3 - neural network for object detection},
    month        = mar,
    year         = 2019,
    doi          = {10.5.../zenodo.1....},
    version      = {1.0},
    publisher    = {Zenodo},
    url          = {https://doi.org/10.5.../zenodo.1....}
    }
```

----

 https://github.com/AASJournals/Tutorials/blob/master/Repositories/CitingRepositories.md

> We recommend inline parenthetical or footnote markup to add direct links to non-static codebases or data:
 
`\citet[][]{alexeyab84_darknet_2019_15...}\footnote{Codebase: \url{https://github.com/AlexeyAB/darknet} }`

`\citep[][Codebase: \url{https://github.com/AlexeyAB/darknet}]{alexeyab84_darknet_2019_15...}`

`\citep[][Dataset: \url{https://doi.org/10.5.../zenodo.15...}]{lia_corrales_2019_15...}`

**Comment by cenit** [comment]
Zenodo already gives you bibitem and citations in many different formats, automatically and without any additional work, for every release produced. I was able to use them without any problem in peer reviewed journals, that’s why I was suggesting Zenodo. I hope it could work also for @HamzahNizami

**Comment by dselivanov** [comment]
@AlexeyAB is this still valid?
```
@misc{alexeyab84_darknet,
  author = {Redmon, J., Bochkovskiy, A., Sinigardi, S.},
  title = {Darknet: Yolov3 - neural network for object detection},
  year = {2019},
  publisher = {GitHub},
  journal = {GitHub repository},
  howpublished = {\url{https://github.com/AlexeyAB/darknet}},
  commit = {6231b748c44e2007b5c3cbf765a50b122782c5a2}
}
```

**Comment by AlexeyAB** [comment]
@dselivanov Yes, just `year= {2020}` and `commit = {df9e60281931844ea63a365c920c3b1a4dc8d8e1}`

**Comment by cenit** [comment]
@AlexeyAB shall we do a new release? Latest one is almost one year old...

**Comment by AlexeyAB** [comment]
@cenit Yes, I did: [![DOI](https://zenodo.org/badge/75388965.svg)](https://zenodo.org/badge/latestdoi/75388965)

So how Darknet should be cited now?

**Comment by cenit** [comment]
it's better to add the DOI to the citation
That number is indexed by all systems, and used as a reference when computing citations

In any case, at the bottom right of the https://zenodo.org/record/3693999 page which you reach clicking on the badge (which you perfectly added to the home page!) there is plenty of examples of proper citations: bibtex, datacite, json, ...

**Comment by AlexeyAB** [comment]
Yes, there are example in the **bottom right corner: Export**
So we can use:
```
  doi          = {10.5281/zenodo.3693999},
  url          = {https://doi.org/10.5281/zenodo.3693999}
```
https://zenodo.org/record/3693999/export/hx#.Xl0vKSEzapo
